pkg/tuf/config.go

```diff
@@ -45,7 +45,7 @@ func (c *Config) Persist(p string) error {
 	if err != nil {
 		return fmt.Errorf("failed to JSON marshal config: %w", err)
 	}
-	err = os.WriteFile(p, b, 0400) // Read only by current user
+	err = os.WriteFile(p, b, 0600)
```

Comment:
@kommendorkapten FYI, I changed this from 0400 to 0600 as future calls to Persist will fail otherwise.

Reply:
Ah, yes of course. Good find!